examples/sqs-simple/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/beatlabs/patron"
+	"github.com/beatlabs/patron/component/async"
+	patronsqs "github.com/beatlabs/patron/component/async/sqs"
+	"github.com/beatlabs/patron/log"
+)
+
+type sqsConfig struct {
+	endpoint string
+	name     string
+	region   string
+}
+
+// Override these values and make sure you have credentials and a session token in your ~/.aws/credentials file.
+// Then set the appropriate value to an AWS_PROFILE env var when running this example.
+// The assumed role needs to have access to the queue.
+var sampleConfig = sqsConfig{
+	endpoint: "https://sqs.eu-west-1.amazonaws.com/315266612443/sandbox-payin",
```

Comment:
We should use in the example the localstack which is available in the docker-compose of the repository and not have the end user provide the parameters

Reply:
Addressed